clippy_dev/Cargo.toml

```diff
@@ -7,10 +7,16 @@ edition = "2018"
 [dependencies]
 bytecount = "0.6"
 clap = "2.33"
+flate2 = "1.0.19"
 itertools = "0.9"
 opener = "0.4"
 regex = "1"
+serde = {version = "1.0", features = ["derive"]}
+serde_json = "1.0"
 shell-escape = "0.1"
+tar = "0.4.30"
+toml = "0.5"
+ureq = "2.0.0-rc3"
```

Comment:
Should we put those deps behind a feature?

Reply:
Added a "crater" feature flag and a "cargo dev-crater" shotcut which builds `cargo dev` with the feature and runs `cargo dev crater`.